support-frontend/assets/pages/supporter-plus-landing/setup/threeTierABTest.ts

```diff
@@ -1,7 +1,20 @@
 import type { Participations } from 'helpers/abTests/abtest';
 
-export const inThreeTierV2Variant = (
+export const showThreeTierCheckout = (
 	abParticipations: Participations,
 ): boolean => {
-	return abParticipations.threeTierCheckoutV2 === 'variant';
+	return (
+		inThreeTierV3Fixed(abParticipations) ||
+		inThreeTierV3Variable(abParticipations)
+	);
+};
+
+const inThreeTierV3Fixed = (abParticipations: Participations): boolean => {
```

Comment:
What do you think to renaming these exports:

`inThreeTierV3Fixed` ---> `showThreeTierFixedPrice`
`inThreeTierV3Variable` ----> `showThreeTierVariablePrice`

I feel these names are a bit more descriptive of the intention of the function and would make the code easier to read / follow, plus they wouldn't need updating if the test name changes.

Reply:
Understand, makes sense, V3 probably does not mean much if you don't know about Test3 (and why would you just looking at code), updated